scripts/lay-down-os

```diff
@@ -143,11 +137,8 @@ if [ -n ${ENV} ]; then
             ;;
         "rancher-upgrade")
             mount_device 
-            if [ -e ${BASE_DIR}/boot/grub/grub.cfg ]; then
-                grub2_config
-            else
-                pvgrub_config
-            fi
+            grub2_config
+            pvgrub_config
```

Comment:
Is there are strong reason to throw in both file types on an upgrade? It could cause confusion for the user who goes looking.